scripts/configs/active_sampler_learning.yaml

```diff
@@ -21,44 +21,49 @@ APPROACHES:
     FLAGS:
       explorer: "active_sampler"
       active_sampler_explore_task_strategy: "random"
-  random_nsrts_explore:
-    NAME: "active_sampler_learning"
-    FLAGS:
-      explorer: "random_nsrts"
+  # random_nsrts_explore:
+  #   NAME: "active_sampler_learning"
+  #   FLAGS:
+  #     explorer: "random_nsrts"
 ENVS:
   grid_row:
     NAME: "grid_row"
-  regional_bumpy_cover:
-    NAME: "regional_bumpy_cover"
-    FLAGS:
-      regional_bumpy_cover_include_impossible_nsrt: "True"
-      bumpy_cover_num_bumps: 3
-      bumpy_cover_spaces_per_bump: 5
-      bumpy_cover_init_bumpy_prob: 0.25
-      cover_num_blocks: 10
-      cover_block_widths: '[0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01]'
-      cover_num_targets: 10
-      cover_target_widths: '[0.008,0.008,0.008,0.008,0.008,0.008,0.008,0.008,0.008,0.008]'
+  sticky_table:
+    NAME: "sticky_table"
+  sticky_table_tricky_floor:
+    NAME: "sticky_table_tricky_floor"
+  # regional_bumpy_cover:
+  #   NAME: "regional_bumpy_cover"
+  #   FLAGS:
+  #     regional_bumpy_cover_include_impossible_nsrt: "True"
+  #     bumpy_cover_num_bumps: 3
+  #     bumpy_cover_spaces_per_bump: 5
+  #     bumpy_cover_init_bumpy_prob: 0.25
+  #     cover_num_blocks: 10
+  #     cover_block_widths: '[0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01]'
+  #     cover_num_targets: 10
+  #     cover_target_widths: '[0.008,0.008,0.008,0.008,0.008,0.008,0.008,0.008,0.008,0.008]'
```

Comment:
can we just not comment out things and have everything in this file uncommented when we're PR'ing to master? That way, things don't change from PR to PR